cmd/etcd.go

```diff
@@ -150,24 +144,68 @@ func putKeyCommandFunc(cmd *cobra.Command, args []string) {
 
 	reqData, err := json.Marshal(putParameter)
 	if err != nil {
-		cmd.Printf("Failed to put key: %v\n", err)
+		cmd.Println(errors.Wrap(err, "Failed to put key"))
 		return
 	}
 	req, err := getRequest(putPrefix, http.MethodPost, "application/json",
 		bytes.NewBuffer(reqData))
 	if err != nil {
-		cmd.Printf("Failed to put key: %v\n", err)
+		cmd.Println(errors.Wrap(err, "Failed to put key"))
 		return
 	}
 	res, err := dail(req)
 	if err != nil {
-		cmd.Printf("Failed to put key: %v\n", err)
+		cmd.Println(errors.Wrap(err, "Failed to put key"))
 		return
 	}
 
 	cmd.Println(res)
 }
 
+func evictOwnerCommandFunc(cmd *cobra.Command, args []string) {
+	cmd.Println("[DANGEROUS] Are you sure to evict the owner?yes/no.")
+	confirm, err := askForConfirmation()
+	if !confirm {
+		return
+	}
+	if err != nil {
+		cmd.Println(errors.Wrap(err, "Failed to evict"))
+		return
+	}
+
+	res, err := getDDLInfo()
+	if err != nil {
+		cmd.Println(errors.Wrap(err, "Failed to evict"))
+		return
+	}
+	var jsn ddlInfoResponse
+	err = json.Unmarshal([]byte(res), &jsn)
+	if err != nil {
+		cmd.Println(errors.Wrap(err, "Failed to evict"))
+		return
+	}
+	var ddlOwnerKey string
+	var ddlOwnerSchemaVersionKey string
+	for _, v := range jsn.Kvs {
+		if strings.HasPrefix(v["key"], ddlOwnerKeyPrefix) {
+			ddlOwnerKey = v["key"]
+			ddlOwnerSchemaVersionKey = ddlAllSchemaVersionsPrefix + v["value"]
+			break
+		}
+	}
+
+	_, err = delKey(ddlOwnerKey)
+	if err != nil {
+		cmd.Println(errors.Wrap(err, "Failed to evict"))
```

Comment:
fmt.Sprintf("Failed to delete the key:%v", ddlOwnerKey)

Reply:
`Sprintf`?